server/index.js

```diff
@@ -1,21 +1,29 @@
 import {setupApp} from './setup-app';
 import {createPageConfig} from './model/page-config';
+import {UiComponent} from './model/ui-component';
+import {Picture} from './model/picture';
 import {default as filtersMap} from './filters';
+import Prismic from 'prismic-javascript';
 import {
   getPrismicApi,
   getEvent
 } from './services/prismic';
 import {
   parsePromoListItem,
-  parseExhibitionsDoc
+  parseExhibitionsDoc,
+  prismicImage,
+  asText
 } from './services/prismic-parsers';
 
 export {setupApp};
 export const filters = filtersMap.toJS();
-export const model = {createPageConfig};
+export const model = {createPageConfig, Picture, UiComponent};
+export {Prismic};
```

Comment:
I was trying to keep Prismic knowledge in the Prismic service: https://github.com/wellcometrust/wellcomecollection.org/blob/master/server/index.js#L20

What do you think?